.ci/manage-aur.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+# shellcheck disable=2153
+set -euo pipefail
+
+# This script is used to manage a packages corresponding AUR repo
+# No point in running this script if we don't have keys available
+# we are skipping debug output here to not leak the key
+set +x
+if [[ -z ${AUR_KEY+x} ]]; then
+    echo "No AUR SSH key available, backing off!"
+    exit 1
+else
+    echo "$AUR_KEY" >/app/key_aur
+    chmod 600 /app/key_aur
+fi
+set -x
+
+declare -a PACKAGES
+PACKAGES=("$@")
+
+# CI-specific variables
+if [ -v GITLAB_CI ]; then
+    _CI_PIPELINE_URL="$CI_PIPELINE_URL"
+    _CI_REPOSITORY_URL="$CI_PROJECT_URL"
+elif [ -v GITHUB_ACTIONS ]; then
+    _CI_PIPELINE_URL="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+    _CI_REPOSITORY_URL="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/commits/main"
+else
+    echo "Supplying pipeline/project details in commit messages is only supported on GitLab CI/GitHub Actions."
+fi
+
+
+# shellcheck source=/dev/null
+source .ci/util.shlib
+export GIT_SSH_COMMAND="-i /app/key_aur ssh -o StrictHostKeyChecking=accept-new"
+
+[[ -v "TMPDIR" ]] || TMPDIR="/tmp"
+mkdir -p "$TMPDIR/aur-push"
+
+if [ -v "PACKAGES[0]" ] && [ "${PACKAGES[0]}" == "all" ]; then
+    echo "AUR push of all managed packages requested."
+    UTIL_GET_PACKAGES PACKAGES
+fi
+
+# Check if the array of packages is empty
+if [ ${#PACKAGES[@]} -eq 0 ]; then
+    echo "No packages to push."
+    exit 0
+fi
+
+for package in "${PACKAGES[@]}"; do
+    unset VARIABLES
+    declare -A VARIABLES
+    if UTIL_READ_MANAGED_PACAKGE "$package" VARIABLES; then
+        if [[ -v "VARIABLES[CI_MANAGE_AUR]" ]] && [[ "${VARIABLES[CI_MANAGE_AUR]}" != "true" ]]; then
+            continue
+        else
+            echo "Starting AUR management.."
+
+            # Clone via SSH to allow pushing
+            if ! test -d "$TMPDIR/aur-push/$package"; then
+                git clone "ssh://aur@aur.archlinux.org/$package.git" "$TMPDIR/aur-push/$package"
+            fi
+
+            # We always run shfmt on the PKGBUILD. Two runs of shfmt on the same file should not change anything
+            shfmt -w "$TMPDIR/aur-pulls/$package/PKGBUILD"
+
+            # Rsync: delete files in the destination that are not in the source. Exclude deleting .CI, exclude copying .git
```

Comment:
that kinda makes no sense

```suggestion
            # Rsync: delete files in the destination that are not in the source. Exclude copying .CI and .git
```